pithos/plugins/lastfm.py

```diff
@@ -100,13 +100,15 @@ def send_rating(self, song, rating):
             logging.info("Sending song rating to last.fm")
 
     def scrobble(self, song):
-        duration = song.get_duration_sec()
-        position = song.get_position_sec()
-        if not song.is_ad and duration > 30 and (position > 240 or position > duration/2):
-            logging.info("Scrobbling song")
-            mode = self.pylast.SCROBBLE_MODE_PLAYED
-            source = self.pylast.SCROBBLE_SOURCE_PERSONALIZED_BROADCAST
-            self.worker.send(self.scrobbler.scrobble, (song.artist, song.title, int(song.start_time), source, mode, duration, song.album))
+        if song.is_ad == False:
+            duration = song.get_duration_sec()
+            position = song.get_position_sec()
+            if duration and position is not None:
```

Comment:
Currently it does `if duration` *and* `if position is not None`. In this case I there is no real issue with not comparing to `None`. Just letting you know that doesn't actually work how english might expect it to.